docs/decisions/0013-special-exam-submission-and-review-events.rst

```diff
@@ -0,0 +1,37 @@
+12. Event definitions for special exam post-submission and review
+#################################################################
+
+Status
+******
+
+**Draft** 2023-09-29
+
+Context
```

Comment:
I think the context section might be a good place to explain the meaning of exam / special exam.

Rough points to hit
- course sections with an `exam_type` have additional logic that governs completion, grading, credit requirements, and more based on the type of exam.
- these sections are sometimes referred to as special exams
- course sections without an exam type configured may still be graded as an exam. This type of content does not have the exam user experience and is not governed by any exam specific logic.